tests/Context/Video/Module/Video/Application/Update/UpdateVideoTitleTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace CodelyTv\Test\Context\Video\Module\Video\Application\Update;
+
+use CodelyTv\Context\Video\Module\Video\Application\Update\UpdateVideoTitleCommandHandler;
+use CodelyTv\Context\Video\Module\Video\Application\Update\VideoTitleUpdater;
+use CodelyTv\Context\Video\Module\Video\Domain\VideoId;
+use CodelyTv\Shared\Domain\ValueObject\Uuid;
+use CodelyTv\Test\Context\Video\Module\Video\Application\Find\FindVideoQueryStub;
+use CodelyTv\Test\Context\Video\Module\Video\Domain\VideoIdStub;
+use CodelyTv\Test\Context\Video\Module\Video\Domain\VideoStub;
+use CodelyTv\Test\Context\Video\Module\Video\Domain\VideoTitleStub;
+use CodelyTv\Test\Context\Video\Module\Video\VideoModuleUnitTestCase;
+
+final class UpdateVideoTitleTest extends VideoModuleUnitTestCase
+{
+    /** @var UpdateVideoTitleCommandHandler */
+    private $handler;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $updater = new VideoTitleUpdater($this->repository());
+
+        $this->handler = new UpdateVideoTitleCommandHandler($updater);
+    }
+
+    /** @test */
+    public function it_should_update_a_video_title()
+    {
+        $query = FindVideoQueryStub::random();
```

Comment:
There's no need to create a `FindVideoQuery`. We would be only using it for the `$id = VideoIdStub::create($query->id());` assignment, which could be done using `VideoIdStub::random`.

In fact, we're only using the `VideoId` in order to create the `$video = VideoStub::withId($id);`. We could do so directly calling to `VideoStub::random`, and later on, in the `$updateVideoTitleCommand` assignment, we could use the `$video->id()` getter in order to get the ID without even needing to interact with the `VideoIdStub` 🤟

This other way we wouldn't be coupling this test to the `FindVideo` use case 🙂